larky/src/main/java/com/verygood/security/larky/parser/StarlarkUtil.java

```diff
@@ -162,7 +162,8 @@ public static Object valueToStarlark(Object x, @Nullable Mutability mutability)
         Map<?, ?> map = (Map<?,?>) x;
         if (!map.isEmpty() && map.values().iterator().next() instanceof List) {
           // Recursively convert subelements.
-          Dict<Object, Object> dict = Dict.of(null);
+          Mutability mu = Mutability.IMMUTABLE;
```

Comment:
Can you explain your reasoning here? `Dict.of(null)` defaults to an immutable dict.

Reply:
sorry just an accident, reverting